src/compute/src/logging/compute.rs

```diff
@@ -373,19 +401,44 @@ struct DemuxState<A: Allocate> {
     peek_stash: BTreeMap<Uuid, Duration>,
     /// Arrangement size stash
     arrangement_size: BTreeMap<usize, ArrangementSizeState>,
+    /// Maps error-logging exports to exports they forward errors to.
+    error_forwards: BTreeMap<GlobalId, BTreeSet<GlobalId>>,
```

Comment:
This has since be removed, but my rationale was the it's possible to drop a dependency without dropping the dataflows that depend on it, and I wanted to keep the forwarding working even when the index that errors are forwarded from has already been dropped. Although that wouldn't have been strictly necessary, because the compute controller never drops dependencies when dependents still exist (but the replica code could handle that I think).